app/src/common/shared/com/igalia/wolvic/browser/LoginStorage.kt

```diff
@@ -22,7 +23,11 @@ class LoginStorage(
     init {
         EngineProvider.getOrCreateRuntime(context).setUpLoginPersistence(places.logins)
         GlobalScope.launch(Dispatchers.IO) {
-            places.logins.value.warmUp()
+            try {
+                storage.value.warmUp()
+            } catch (e: InvalidKeyException) {
+                storage.value.wipeLocal()
```

Comment:
This wipes the whole local storage. Have you tried to get rid of the login database only?

Reply:
Note that `storage` equals `places.logins`, see <https://github.com/mozilla-mobile/android-components/tree/main/components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa>.

As far as I understand it wipes only the logins database.

Reply:
yeah sorry, I missread the patch. I shouldn't review while on a meeting :smile: